draft-barnes-acme.md

```diff
@@ -1861,12 +1859,12 @@ The client serializes the validation object to UTF-8, then uses its account
 private key to sign a JWS with the serialized JSON object as its payload.  This
 JWS is NOT REQUIRED to have the "nonce" header parameter.
 
-The record provisioned to the DNS is the "signature" value from the JWS, i.e.,
-the base64-encoded signature value.  The client constructs the validation domain
-name by appending the label "_acme-challenge" to the domain name being
-validated, then provisions a TXT record with the signature value under that
-name. For example, if the domain name being validated is "example.com", then the
-client would provision the following DNS record:
+The record provisioned to the DNS is the base64-encoded SHA-256 of the JWS
```

Comment:
The signature does get verified.  Step 1 verifies the JWS.  Step 5 verifies that the JWS (from step 1) is located in the DNS record.

It makes sure that the ACME client has control of the account key, and the authorization (DNS entry/certificate) is explicitly linked to the validation response.